waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -105,10 +105,7 @@ npmDepsForWasp _spec =
             ("react-dom", "^17.0.2"),
             ("react-query", "^3.39.2"),
             ("react-router-dom", "^5.3.3"),
-            ("react-scripts", "4.0.3"),
-            -- NOTE: We need to specify this exact version of `react-error-overlay` for use with
-            -- `react-scripts` v4 due to this issue: https://github.com/facebook/create-react-app/issues/11773
-            ("react-error-overlay", "6.0.9")
+            ("react-scripts", "5.0.1")
```

Comment:
Should we pin to exact version, or allow patch updates?

Reply:
Hm, we had it pinned before, but I wasn't sure why so I checked the history. It has been pinned down from [the very start](https://github.com/wasp-lang/wasp/commit/b43bfdfbfeba0e170061ed23bc8c2260fc56bb2e#diff-1303ebcff061f33c31aef11b7b7422d3826ffdabebbaad0a6be281b9051ee54c). Whenever we'd update a version, we'd leave it pinned. I wasn't sure why so I created an empty project with create-react-app and, sure enough, they pin it down too (I'm guessing Martin just copied what CRA generated at the time:
```json
  "dependencies": {
    "@testing-library/jest-dom": "^5.16.5",
    "@testing-library/react": "^13.4.0",
    "@testing-library/user-event": "^13.5.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-scripts": "5.0.1",
    "web-vitals": "^2.1.4"
  },
```
Anyway, I think it's best we keep it pinned down.

Reply:
Yup, I must have just kept it as it was.
Hm, I don't know! I did quick googling and it is unclear. We pin down, we avoid possible hiccups. We don't pin down, we allows for possible fixes. Let's leave it pinned down I guess, if that is what they recommended.

Reply:
Sounds good, thanks for confirming gang. Will leave it as-is. 👍🏻